scheduler/src/cook/compute_cluster.clj

```diff
@@ -17,6 +17,25 @@
   (:require [clojure.tools.logging :as log]
             [datomic.api :as d]))
 
+(defprotocol ComputeCluster
+  ; These methods should accept bulk data and process in batches.
+  ;(kill-tasks [this task]
+  ;(launch-tasks [this offers task-metadata-seq]
+  (compute-cluster->db-id [this])
+
+  (get-mesos-driver-atom-hack [this]
```

Comment:
Can we have this return the driver instead of an atom? I think in an ideal world we'd initialize all of the things that need a driver when we take leadership instead of relying on the atom. I'd rather keep those referencing the atom explicitly for now than stick the atom everywhere else.

Reply:
Did this change.